preprod.healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -2,209 +2,67 @@
+    "name": "healworkspaces",
+    "region": "us-east-1"
+  },
+  "pay-models": [
     {
       "user_id": "cgmeyer@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
+      "aws_account_id": "134208290168",
+      "name": "strides_grant",
+      "region": "us-east-1",
+      "ecs": "true",
+      "subnet": 0
     },
     {
       "user_id": "xritter@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "planxdemo@gmail.com",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
+      "aws_account_id": "134208290168",
+      "name": "strides_grant",
+      "region": "us-east-1",
+      "ecs": "true",
+      "subnet": 2
     },
     {
-      "user_id": "hughestr@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "tara@taralichtenberg.com",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "trevars@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "qureshi@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "mbkranz@gmail.com",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "jmontmaxwell@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "rgrossman1@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "philschumm@gmail.com",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "Kranz-michael@norc.org",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "0000-0002-0093-3367",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "tentner-andrea@norc.org",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
+      "user_id": "egiger@uchicago.edu",
+      "aws_account_id": "134208290168",
+      "name": "strides_grant",
+      "region": "us-east-1",
+      "ecs": "true",
+      "subnet": 3
     },
     {
-      "user_id": "LeClere-Felicia@norc.org",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
```

Comment:
also FL should stay

Reply:
The default paymodel will take care of everyone that are not specified under paymodels.